src/hdtDefaultBBP.cpp

```
@@ -11,28 +17,110 @@ namespace hdt
+		std::string readLooseFile(const std::filesystem::path& path)
+		{
+			std::ifstream f(path, std::ios::binary);
+			if (!f)
+				return {};
+			std::ostringstream ss;
+			ss << f.rdbuf();
+			return ss.str();
+		}
+	}
+
 	void DefaultBBP::loadDefaultBBPs()
```

Clang (14.0.6)

[warning] method 'loadDefaultBBPs' can be made static

(readability-convert-member-functions-to-static)